emojivoto-voting-svc/cmd/server.go

```diff
@@ -21,15 +27,43 @@ func main() {
 		log.Fatalf("GRPC_PORT (currently [%s]) environment variable must me set to run the server.", grpcPort)
 	}
 
+	if promPort == "" {
```

Comment:
what do you think about simply skipping the prometheus server altogether if this is not configured, rather than failing?